sqlparser/lib/src/engine/module/fts5.dart

```diff
@@ -122,7 +122,8 @@ class Fts5Table extends Table {
   }) : super(
           name: name,
           resolvedColumns: [
-            if (contentTable != null && contentRowId != null) RowId(),
+            if (contentTable != null && contentRowId != null)
+              RowId(includedInResults: true),
```

Comment:
This will also have the effect of including the `rowid` column when expanding `*` columns. From what I can see, that's not correct though:  `SELECT * FROM fts_table_with_external_content` does not include the rowid column, I need to be explicit and say `SELECT rowid, * FROM ...`.

Do you perhaps have a statement that doesn't work at the moment as an example? Maybe we need to expose this rowid column in a different way.

Reply:
I need this column to insert entries into the fts table, where I need to address the rowid colum directly like:
```sql
INSERT INTO fts(rowid, name) VALUES(0, 'test');
``

Reply:
It looks like drift is able to resolve the rowid correctly though, this statement is analyzed fine:
https://github.com/simolus3/drift/blob/039ff942f77888adce75b406234ecb4fdbd60264/examples/modular/lib/src/search.drift#L16-L18

Reply:
Yes, I also use a trigger `ON DELETE` on my content table and that works for the analyzer. For the insertion I use dart logic to insert the entries manually into the fts table via the generated `DataClass`. There the `rowid` is only available when setting the `includedInResults` to `true`. Sorry for my unclear previous answer.